api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirObservationDaoImpl.java

```diff
@@ -40,6 +55,36 @@
 	@Autowired
 	private ObservationCategoryMap categoryMap;
 	
+	@Autowired
+	@Getter(AccessLevel.PUBLIC)
+	@Setter(AccessLevel.PUBLIC)
+	@Qualifier("sessionFactory")
+	private SessionFactory sessionFactory;
+	
+	@Override
+	@Authorized(PrivilegeConstants.GET_OBS)
```

Comment:
This is already annotated in the interface definition. We can remove it from here.